spec/integration/v2/tailorings_spec.rb

```diff
@@ -207,4 +207,93 @@
       end
     end
   end
+
+  path '/policies/{policy_id}/tailorings/{tailoring_id}/rules/{id}' do
+    let(:canonical_profile) do
+      FactoryBot.create(
+        :v2_profile,
+        rule_count: 3,
+        supports_minors: [1]
+      )
+    end
+
+    let(:policy_id) do
+      FactoryBot.create(
+        :v2_policy,
+        account: user.account,
+        profile: canonical_profile,
+        supports_minors: [1]
+      ).id
+    end
+
+    let(:tailoring_id) do
+      FactoryBot.create(
+        :v2_tailoring,
+        :with_mixed_rules,
+        profile: canonical_profile,
+        policy: V2::Policy.find(policy_id),
+        os_minor_version: 1
+      ).id
+    end
+
+    patch 'Edit Tailoring Rules' do
+      let(:item) { FactoryBot.create(:v2_rule, security_guide: canonical_profile.security_guide) }
+
+      v2_auth_header
+      tags 'Policies'
+      description 'Assign a Rule to a Tailoring'
+      operationId 'AssignRule'
+      content_types
+
+      parameter name: :policy_id, in: :path, type: :string, required: true
+      parameter name: :tailoring_id, in: :path, type: :string, required: true
+      parameter name: :id, in: :path, type: :string, schema: { type: 'string', format: 'uuid' }, required: true
```

Comment:
Consistency with systems:
```suggestion
      parameter name: :id, in: :path, type: :string, required: true
```